prepare_env/example_data.py

```
@@ -0,0 +1,102 @@
+# sudo python -m pip install pymongo
+# run Openvillage api server before running this script!
+import json
+import random
+import requests
+import string
+
+from pymongo import MongoClient
+
+SERVER_URL = 'http://localhost:8080'
+DB_URL = 'mongodb://localhost:27017/'
+PASSWORD_FOR_ALL_USERS = 'qazwsx'
+
+
+def register_user(username, email, full_name):
+    payload = {'username': username,
+               'password': PASSWORD_FOR_ALL_USERS,
+               'email': email,
+               'fullname': full_name,
+               'birthdate': "02/02/1992"}
+    r = requests.post(SERVER_URL + "/api/users/add", data=payload)
+    print "Adding user " + username + "(" + email + "): " + str(r.status_code)
+    if r.status_code != 200:
+        print r.text
```

If you cannot create user next steps don't make a sense. Please raise an Exception in this place.